Meta/lint-shell-scripts.sh

```diff
@@ -34,7 +34,7 @@ if (( ${#files[@]} )); then
         exit 1
     fi
 
-    shellcheck "${files[@]}"
+    shellcheck -P "./Meta/" "${files[@]}"
```

Comment:
This works on ShellCheck 0.7.2 but seems to fail on CI.

Reply:
CI is using shellcheck-0.7.0-2build2 it seems https://github.com/actions/virtual-environments/blob/ubuntu20/20220405.4/images/linux/Ubuntu2004-Readme.md#installed-apt-packages

Seems worth adding shellcheck -v to the`Check versions` job in .github/workflows/cmake.yml

And possibly trying to apt install an updated version of it? Not sure where 0.7.2+ would be found when we're using github's provided 20.04 VMs.

Reply:
Looks like we will need to add the [latest shellcheck binary to the CI build image](https://github.com/koalaman/shellcheck#installing-a-pre-compiled-binary=). I will take a look at this and making sure the `Check versions` also checks the version of shellcheck.

Is it a problem if people running Ubuntu 20.04 can't get the correct ShellCheck version from apt?

Reply:
I found in the past that you simply can't rely on the shellcheck version in any repositiory. So all my CIs are either using koalaman/shellcheck-alpine docker container or getting a up-to-date binary from koalamans GitHub.

Reply:
Since we want the linting to work correctly on dev machines and I don't want to force users to install ShellCheck through non-package manager ways (as the developer experience at the moment is excellent) I will just disable the error from the two places it happens.